frontend/src/ProjectManager/ProjectManagerScriptOptions.svelte

```diff
@@ -58,10 +53,37 @@
       return await r.json();
     });
   }
+
+  $: showScriptPromise = fetch(
+    `${$settings.backendUrl}/get_project_script?project=${$selectedProject.session_id}&script=${args.name}`
+  ).then(async (r) => {
+    if (!r.ok) {
+      throw Error(r.statusText);
+    }
+    return (await r.text()).split("\n");
+  });
+
+  let toolbarButtons = [
+    {
+      text: "Delete Script",
+      iconUrl: "/icons/trash.svg",
+      shortcut: "D",
+      onclick: () => {
+        deleteScript;
+      },
+    },
+  ];
 </script>
 
-<div>
-  <button on:click|stopPropagation="{deleteScript}"
-    ><Icon url="/icons/trash.svg" />Delete {args.name} from project.</button
-  >
+<div class="hbox">
+  <div class="toolbar">
+    <Toolbar toolbarButtons="{toolbarButtons}" />
+  </div>
+  <div class="script">
+    {#await showScriptPromise}
+      <LoadingAnimation />
+    {:then loadedScript}
+      <Script script="{loadedScript}" />
```

Comment:
This script pane goes all the way to the bottom of the window which looks a little weird, it would probably look better with a tiny bit of bottom padding.

Reply:
yeah true ill cut it off at the version number